src/dotnet/commands/dotnet-publish/LocalizableStrings.resx

```diff
@@ -155,4 +155,10 @@ The default is to publish a framework-dependent application.</value>
   <data name="CmdNoLogo" xml:space="preserve">
     <value>Do not display the startup banner or the copyright message.</value>
   </data>
+  <data name="NoSelfContainedOptionDescription" xml:space="preserve">
+    <value>Publish your application as a framework dependent application without the .NET Core runtime. A supported .NET Core runtime must be installed to run your application.</value>
+  </data>
+  <data name="SelfContainAndNoSelfContainedConflict" xml:space="preserve">
+    <value>Self contained and no self contained arguments are conflicting.</value>
```

Comment:
Let's use the option names in the message.  How about:

```suggestion
    <value>The '--self-contained' and '--no-self-contained' options conflict with each other.  Specify only one of the options.
```

cc @KathleenDollard for additional input.

Reply:
This updated message looks good